### PR TITLE
[13.0][ADD] l10n_th_check_scb_bank

### PR DESCRIPTION
**Pull request by Saran440**
Add form pre-printed SCB check paper.
- Amount text in SCB Check depend on currency payment.

this module depend on
- [x] l10n_th_amount_to_text : https://github.com/OCA/l10n-thailand/pull/139
- [x] account_check_date : https://github.com/OCA/account-payment/pull/335
- [x] account_check_printing_report_base : https://github.com/OCA/account-payment/pull/351

### CODE REVIEW

**Comment by newtratip** [comment]
Hello @Saran440 Do you have a plan to continue this ?

**Review by ps-tubtim — APPROVED** [review]
Code, Functional Test: OK.

File: l10n_th_check_scb_bank/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "account_check_printing_report_base",
+        "account_check_date",
+    ],
+    "data": [
+        "data/account_payment_check_report_data.xml",
+        "data/report_data.xml",
+        "report/report_check_scb_base.xml",
+    ],
+    "installable": True,
+    "development_status": "alpha",
```
Comment (newtratip) [inline l10n_th_check_scb_bank/__manifest__.py:23]:
```suggestion
    "development_status": "Alpha",
```

**Review by newtratip — COMMENTED** [review]
Some review.

**Comment by newtratip** [comment]
@Saran440 I can not print SCB Check, this is step to error.

1. Create vendor bill
2. Registor payment
3. Print check >> I don't see any data in form.

Selection_490

My video
Peek 2020-11-10 14-32

I removed font THSarabunNew in the form, I can print correctly. I don't sure this is cause to the error.

wkhtmltopdf version : 0.12.4

**Pull request closed without merging**